.kokoro/samples-test.sh

```diff
@@ -21,7 +21,6 @@ export NPM_CONFIG_PREFIX=/home/node/.npm-global
 # Setup service account credentials.
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
 export GCLOUD_PROJECT=long-door-651
-source ${KOKORO_GFILE_DIR}/tasks-service-account.sh
```

Comment:
@jkwlui @averikitsch not entirely sure what's happening here :) But it looks like this change and autosynth are gonna fight.

Reply:
The sample needed a service account email (name@projectID.iam.gserviceaccount.com) for it to make a request. I added it to the Kokoro bucket so I could load it from the environment variable in the test (https://github.com/googleapis/nodejs-tasks/blob/master/samples/test/test.samples.js#L26).

Reply:
I discussed with other teammates and we don't see an issue removing this and just adding the service account email to the test. Would you like me to do that?

Reply:
Please do!  That sounds fine to me.